ajax/libs/goalProgress/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "goalProgress",
+  "filename": "goalProgress.min.js",
+  "version": "1.0",
+  "description": "Animated progress bar using jQuery to show how close you are to reaching your goal.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinacious/goalProgress.git"
+  },
+  "keywords": [
+    "goalProgress",
+    "tags",
+    "jQuery",
+    "animated"
+  ],
+  "author": "tinacious",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tinacious/goalProgress.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Let's modify the pattern as `"goalProgress*"`
because this pattern can get the correct files via auto-update config when new version release.
Thank you.

Reply:
Hello @sufuf3 I am unsure what you mean in this comment, could you please clarify

Reply:
`basePath` is the path in the npmjs tarball or GitHub repo; it will be ignored when files are copied to cdnjs.

`files` indicates the file(s) to copy and can be named (e.g., lodash.min.js) or wildcards (e.g., *.js).

Auto-update mechanism will get files via these fields.

So in this case, when there is release new version, we want to get `"goalProgress*"` files, we have to fill in this pattern in `"files"` field and get these files which under `""` path, so we need to fill `""` into `"basePath"` field.

You could refer to https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md#23-auto-update-syntax

Thank you.